com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/flowable/SetRetryPhaseAdditionalProcessAction.java

```diff
@@ -0,0 +1,35 @@
+package com.sap.cloud.lm.sl.cf.core.flowable;
+
+import javax.inject.Inject;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class SetRetryPhaseAdditionalProcessAction implements AdditionalProcessAction {
+
+    private static final String RETRY_STEP_PHASE = "RETRY";
+    private static final String STEP_PHASE = "stepPhase";
```

Comment:
Here we define the same constant for the second time. This can be a problem in the future.

Reply:
I had doubts in myself about this and moving the constant in the cf.core project did not seem to be a good idea for me?